### PR TITLE
Minor bug

### PR DESCRIPTION
**Pull request by CraigglesO**
Not a huge issue, just noticed as I am still learning from the module and felt like contributing.

First case of `splitTile` the value `cz` is null.
Still `const m = 1 << (cz - z);` is called, which creates random values, so it will _probably_ never fail, but it was an easy fix, so I thought I would just report it.

Was fun reading your module along with vt-pbf, I get how vector tiles work now.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "geojson-vt",
-  "version": "3.2.1",
+  "version": "3.2.2",
```
Comment (mourner) [inline package.json]: Can you remove this change? I prefer to do it as a part of a separate commit (automatically made with `npm version`) when releasing a new version.

Comment (CraigglesO) [reply package.json]: Oh shoot, my bad. Habits die hard.
